### PR TITLE
build: watch for sauce service failures when waiting for it to start

### PR DESCRIPTION
**Pull request by gregmagolan**
Also build: fix saucelabs jobs that were broken by bazelisk change.

Also add two targets to make it more convenient to tail & dump the sauce service logs: //tools/saucelabs:sauce_service_tail & //tools/saucelabs:sauce_service_log

@kara This will fix the issue you ran into when you were using the wrong sc binary :)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by gregmagolan** [comment]
Saucelabs jobs are normally monitor only so CI is turned on here: https://github.com/angular/angular/pull/36111

File: tools/saucelabs/sauce-service.sh [hunk 1]
```diff
@@ -191,16 +191,31 @@ service-pre-start() {
...
   @echo "Sauce Connect Proxy started (pid $(cat "${SAUCE_PID_FILE}"))"
 }
 
 # Called if service fails to start
 service-failed-setup() {
   if [[ -f "${SERVICE_LOG_FILE}" ]]; then
-    echo "================================================================================"
-    echo "${SERVICE_LOG_FILE}:"
-    echo $(cat "${SERVICE_LOG_FILE}")
+    @echo "tail ${SERVICE_LOG_FILE}:"
```
Comment (josephperrott) [inline tools/saucelabs/sauce-service.sh]: Can we also provide the location of the log file at the end of the output?

**Review by josephperrott — APPROVED** [review]
LGTM

File: tools/saucelabs/sauce-service.sh (hunk 1 above)
Comment (gregmagolan) [reply tools/saucelabs/sauce-service.sh]: 👍 done

**Comment by gregmagolan** [comment]
Wait on green https://github.com/angular/angular/pull/36111

**Review by josephperrott — APPROVED** [review]
LGTM

**Pull request closed without merging**